src/cart/src/cart/crt_rpc.h

```diff
@@ -314,6 +316,7 @@ CRT_GEN_STRUCT(crt_grp_cache, CRT_SEQ_GRP_CACHE)
 
 #define CRT_OSEQ_URI_LOOKUP	/* output fields */		 \
 	((crt_phy_addr_t)	(ul_uri)		CRT_VAR) \
+	((uint32_t)		(ul_tag)		CRT_VAR) \
```

Comment:
Technically this is a protocol change so we should bump the number.

Reply:
good point, will do